### ISSUE TITLE
Simstrat timestep

### ISSUE THREAD

**Issue by f-baerenbold** [aemon-j/LER_examples#17]
Hello,

We usually recommend a timestep of around 300 seconds (5 minutes) for Simstrat in presence of inflows. A timestemp of 3600 seconds (1 hour) as specified in LakeEnsemblR can lead to quite different results.

**Comment by jorritmesman** [comment]
Hello Fabian,

We had added instructions regarding the Simstrat time step in the FAQ document of LakeEnsemblR (https://github.com/aemon-j/LakeEnsemblR/wiki/FAQ) and how to do this. As LakeEnsemblR is created with the mindset of running multiple models in similar ways, we haven't tested if this can be implemented in the configuration file (although the "trick" specified in the FAQ works). Another way would be to run all models with a time step of 300 seconds, of course, but this would lead to larger differences with other models (e.g. the R version of MyLake only runs daily).

Still, I agree this should be possible to implement. I'll check if it's possible to add `Timestep s` under the `model_parameters` section in the configuration file, which, based on the code, seems to be no problem. In this case,  we can add it to at least some examples.

However, in the majority of the cases, Simstrat seems to perform well even with hourly time step.

**Comment by f-baerenbold** [comment]
Hi Jorrit,

Ah, good to see this in FAQ. Could you maybe add that this seems to be an issue only in the presence of in-/outflows?

It would be great to add the 300 s timestep to examples which have in-/outflow.

Thanks,
Fabian

**Comment by jorritmesman** [comment]
Alright, it was indeed possible to set the Simstrat time step from within the LakeEnsemblR.yaml file (by putting `Simulation/Timestep s` in the Simstrat part of the "model_parameters" section). I updated the instructions in the FAQ and added that this is advised when using in-/outflows. I also added these settings to the FictionalReservoir and LakeMendota examples on this repository. 

Thanks a lot for contacting us about this!
